.travis.yml

```diff
@@ -10,3 +10,6 @@ script:
   - gulp run-tests
   - npm run test
   - npm run coveralls
+
+notifications:
+  webhooks: https://coveralls.io/webhook?repo_token=o4Q54byGsbo2r7laPPTHtwjEIQDr5BaXK
```

Comment:
I'm not sure about whether or not it's meant to be hidden... I added this line when my coveralls stopped building, to allow for parallel builds, based on a guide from coveralls... You can check https://coveralls.zendesk.com/hc/en-us/articles/203484329